astroid/brain/brain_builtin_inference.py

```diff
@@ -892,6 +892,26 @@ def _build_dict_with_elements(elements):
     return _build_dict_with_elements([])
 
 
+def _looks_like_copy_method(node):
+    func = node.func
+    return isinstance(func, nodes.Attribute) and func.attrname == "copy"
+
+
+def _has_copy_method(node):
+    return any(
+        isinstance(node, klass)
+        for klass in [nodes.Dict, nodes.List, nodes.Set, objects.FrozenSet]
+    )
```

Comment:
```suggestion
    return isinstance(node, (nodes.Dict, nodes.List, nodes.Set, objects.FrozenSet))
```
I think this work https://docs.python.org/3/library/functions.html#isinstance